extensions/wikia/ArticleVideo/scripts/featured-video.jwplayer.ads.js

```diff
@@ -5,8 +5,9 @@ define('wikia.articleVideo.featuredVideo.ads', [
 	'ext.wikia.adEngine.slot.service.srcProvider',
 	'ext.wikia.adEngine.video.player.jwplayer.adsTracking',
 	'ext.wikia.adEngine.video.vastDebugger',
+	'ext.wikia.adEngine.slot.service.slotRegistry',
```

Comment:
please try to keep alphabetical order in dependencies list when possible ;)

Reply:
Done :)